### PR TITLE
Awesome books: a complete website with navigation

### PR DESCRIPTION
**Pull request by Aduda-Boaz**
This PR is for the complete web page with navigations. We have managed to implement the navigation bars, add the footer and implement luxon to show the current page.
We are happy to present our work for review.
Thank you

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,23 +4,52 @@
...
+      <form class="new-book container w-50 d-flex flex-column align-items-center">
+        <div class="inputs">
+          <input class="ff-caveat fs-5" type="text" minlength="1" maxlength="30" placeholder="Title" id="title" required>
+        </div>
+        <div class="inputs">
+          <input class="ff-caveat fs-5" type="text" minlength="1" maxlength="30" placeholder="Author" id="author" required>
+        </div>
+        <div class="btn float-end">
+          <button type="submit" id="submit">Submit</button>
+        </div>
```
Comment (hemantso) [inline index.html]: - [x] To follow the best practices of HTML, we should use semantic tags like main, section. Let's try to fix this.

File: index.html [hunk 2]
```diff
@@ -4,23 +4,52 @@
...
-<body>
-    <h1>Awesome books</h1>
-    <div class="books">
-
+<body class="container-fluid w-100 d-flex flex-column align-items-center">
+  <nav class="navbar">
+    <a class="nav-link" href="#" id="add">Add new</a>
+    <a class="nav-link" href="#" id="list">List</a>
+    <a class="nav-link" href="#" id="contact">Contact</a>
+  </nav>
```
Comment (hemantso) [inline index.html]: - [x] Your website navigation is working absolutely fine. Let's try to improve the design of the navbar. we can add a logo and nav link on the left and right end of the navbar respectively. Check the example below.

Yours
image

Sample
image

**Review by hemantso — CHANGES_REQUESTED** [review]
Hi @Aduda-Boaz and @JuliCarracedo, 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there! 

Alt Text
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Try to add a `live link` and screenshot of your application in the README file.

Cheers,
Hemant Soni
Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (rloterh) [reply index.html]: This issue is well resolved!✔️

File: index.html (hunk 1 above)
Comment (rloterh) [reply index.html]: This issue is well resolved! ✔️

File: index.html [hunk 3]
```diff
@@ -4,23 +4,54 @@
...
-    <div class="books">
-
+<body class="container-fluid w-100 d-flex flex-column align-items-center">
+  <nav class="navbar container-fluid d-flex justify-content-between">
+    <a class="link-unstyled" href="#"><img alt="logo" src="./Captura.PNG"></a>
+    <div class=" d-flex  justify-content-around">
+      <a class="nav-link" href="#" id="add">Add new</a>
+      <a class="nav-link" href="#" id="list">List</a>
+      <a class="nav-link" href="#" id="contact">Contact</a>
+    </div>
```
Comment (rloterh) [inline index.html:18]: - [ ] OPTIONAL
- For a well structured and cleaner HTML syntax kindly us `ul` and `li` tags for the organizing the menu items
- The right menu elements in your navbar is missing the **dividers** between each menu element, as can be seen in the provided [wireframe design](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md). Kindly ensure to improve upon this aspect in your design, as we really want our designs to as much as possible mimic the provided wireframe template.

**Your design menu:**
image

**The wireframe menu design:**
image

File: index.html [hunk 4]
```diff
@@ -4,23 +4,54 @@
...
+    <h1 class="ff-caveat text-center">Contact information</h1>
+    <p class="text-center">Do you have any question or you want to say "Hello"?
+      <br>
+      You can reach out to us!
+      <br>Our e-mail: email@mail.com <br>Our phone: 0177 223 337 6<br>our address: streetname 2341, County
+    </p>
+
+  </section>
+  <footer class="navbar fixed-bottom d-flex justify-content-center">
+    <a> Corpyiright 2021. All rights reserved.</a>
```
Comment (rloterh) [inline index.html:51]: - [ ] OPTIONAL
- Kindly correct the spelling of `Copyright` 😉  
- Again, as can be seen in the wireframe template, let's ensure that the `Copyright` text is aligned to the left of the footer section

File: index.html [hunk 5]
```diff
@@ -4,23 +4,54 @@
...
+  </section>
+  <section id="contact-div" class="container-fluid d-none">
+    <h1 class="ff-caveat text-center">Contact information</h1>
+    <p class="text-center">Do you have any question or you want to say "Hello"?
+      <br>
+      You can reach out to us!
+      <br>Our e-mail: email@mail.com <br>Our phone: 0177 223 337 6<br>our address: streetname 2341, County
+    </p>
+
+  </section>
```
Comment (rloterh) [inline index.html:49]: - [ ]  OPTIONAL
- The contents of your _contact page_ does match the provided wireframe design template. As we want our design to as much as possible closely follow the the provided [wireframe template](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md) kindly ensure to align `Our email`, ` Our contact` and `Our address` to the **left**, and **bulleted** as can be observed in the template design

**Your design:**
image

**The wireframe design:**
image

**Review by rloterh — APPROVED** [review]
Hi Boaz & Julian, 

Great job for making the necessary changes recommended by the previous reviewer. However, there's always room for improvement.
Kindly consider my OPTIONAL comments

## Optional suggestions
_Please keep in mind that you, for best practices and following the project requirements and the design templates closely kindly consider these aspects as outlined in the below comments._

**_Kindly notice that these are required changes that I'm sorry they didn't show up in the earlier review._**

Also, kindly consider:
- [ ] OPTIONAL:
Kindly note, the correct github flow should be: `merge commits into main from feature_branch`

<br>
Your project is complete! Congratulations! 🥂There is nothing else to say other than... it's time to merge it 👍.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**